lib/compat/wordpress-6.1/class-wp-theme-json-6-1.php

```diff
@@ -808,8 +806,10 @@ function( $pseudo_selector ) use ( $selector ) {
 
 			$has_block_gap_support = _wp_array_get( $this->theme_json, array( 'settings', 'spacing', 'blockGap' ) ) !== null;
 			if ( $has_block_gap_support ) {
-				$block_rules .= '.wp-site-blocks > * { margin-block-start: 0; margin-block-end: 0; }';
-				$block_rules .= ".wp-site-blocks > * + * { margin-block-start: $block_gap_value; }";
+				$block_gap_value = static::get_property_value( $this->theme_json, array( 'styles', 'spacing', 'blockGap' ) );
```

Comment:
Is it worth adding tests in class-wp-theme-json-test.php, or expanding `test_get_stylesheet_generates_layout_styles` for these changes?

Reply:
Great point, yep, I reckon an additional test is a good idea. I'll update 🙂